google/cloud/security/inventory/pipelines/load_projects_iam_policies_pipeline.py

```diff
@@ -99,7 +99,7 @@ def _retrieve(self):
         """
         # Get the projects for which we will retrieve the IAM policies.
         try:
-            project_numbers = self.dao.select_project_numbers(
+            project_numbers = self.dao.get_project_numbers(
```

Comment:
Would this get_project_numbers() better belong to the project_dao?  In other words, would dao become the base_dao?  It's okay to follow-up on this later.